tools/ci/update-vulnerable-dependencies/update-vulnerable-dependencies.sh

```diff
@@ -37,7 +37,7 @@ for dep in $(osv-scanner "${OSV_FLAGS[@]}" | jq -c '.results[].packages[] | .pac
     if [[ "$package" == "stdlib" ]]; then
       go mod edit -go="$fixVersion"
     else
-      go get -u "$package"@v"$fixVersion"
+      go get "$package"@v"$fixVersion"
```

Comment:
We can also use `go get -u=patch` to get only patch release https://pkg.go.dev/cmd/go#hdr-Add_dependencies_to_current_module_and_install_them